documentation/modules/mtv-performance-recommendation.adoc

```diff
@@ -209,3 +210,27 @@ If you are creating many migration plans with a single VM and have enough time b
 * *Maximum transfer rate observed of multiple disks from a single ESXi host: 253 MB/s (concurrent migration of 10 VMs, 1 disk each, 35/50 GiB used per disk)*
 * *Total transfer rate observed of multiple disks (210) from 8 ESXi hosts: 802 MB/s (concurrent migration of 70 VMs, 3 disks each, 35/50 GiB used per disk)*
 
+[id="mtv-precommendations-vms-large-disks_{context}"]
+== Recommendations for migrating VMs with large disks
+
+The following recommendations are suggested for VMs with data on disk totaling to 1 TB or greater for each individual disk:
+
+* Schedule appropriate maintenance windows for migrating large disk virtual machines (VMs). Such migrations are sensitive operations and may require careful planning of maintenance windows and downtime, especially during periods of lower storage and network activity.
+
+* Check that no other migration activities or other heavy network or storage activities are run during those large virtual machine (VM) migrations. You should treat these large virtual machine migrations as a special case. During those migrations, prioritize {project-short} activities. Plan to migrate those VMs to a time when there are fewer activities on those VMs and related datastore.
+
+* For large VMs with a high churn rate, which means data is frequently changed in amounts of 100 GB or more between snapshots, consider reducing the warm migration `controller_precopy_interval` from the default, which is 60 minutes. It is important to ensure that this process is started at least 24 hours before the scheduled cut-over to allow for multiple successful precopy snapshots to complete. When scheduling the cut-over, ensure that the maintenance window allows for sufficient time for the last snapshot of changes to be copied over and that the cut-over process begins at the beginning of that maintenance window.
+
+** Reduce the warm migration `controller_precopy_interval` to 15 minutes. 
+** Schedule the cut-over to be done at least 24 hours before the maintenance window. 
+** Schedule the cut-over to the beginning of that maintenance window. 
+
++
+This schedule should allow the last snapshots to be small enough, including only the changes from the last 15 minutes.
```

Comment:
Remove  those 4 rows  - (as it is a repetition of what is written i the paragraph above and I don[t want to recomend 15 minuts as for some customers we saw 15 munuts is to low, They will need to find the golden interval  that fits them. We just recommend to reduce)

"
**Reduce the warm migration `controller_precopy_interval` to 15 minutes. 
** Schedule the cut-over to be done at least 24 hours before the maintenance window. 
** Schedule the cut-over to the beginning of that maintenance window. 
This schedule should allow the last snapshots to be small enough, including only the changes from the last 15 minutes.
+